flow/flow.cpp

```diff
@@ -220,7 +338,7 @@ int main(int argc, char** argv)
             }
         }
         const auto& phases = Opm::Runspec(*deck).phases();
-
+        bool outputFiles = (outputMode != FileOutputMode::OUTPUT_NONE);
```

Comment:
I think the direction of this (passing the output bools as arguments) is fine, but on this line outputMode is uninitialized if mpiRank != 0.

Reply:
OK - should be fixed now.